spark/src/main/scala/smile/tuning/package.scala

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2010-2020 Haifeng Li. All rights reserved.
+ *
+ * Smile is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * Smile is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Smile.  If not, see <https://www.gnu.org/licenses/>.
+ ******************************************************************************/
+
+package smile
+
+import java.util.function.BiFunction
+
+import org.apache.spark.sql.SparkSession
+import smile.classification.Classifier
+import smile.validation.{Accuracy, ClassificationMeasure, CrossValidation}
+
+import scala.reflect.ClassTag
+
+case class SerializableClassificationMeasure(@transient measure: ClassificationMeasure)
```

Comment:
Please move this to smile.spark package too. It is not good to spread spark code to many packages. Not everyone uses spark.

Reply:
done in smile.spark.tuning, do you prefer smile.spark.tuning or just smile.spark ?